src/components/consul/detail/ConsulCommentListItem.js

```diff
@@ -213,26 +212,25 @@ export const ConsulCommentListItem = ({ commentItem, onRefresh, replyList, navig
         : null}
 
       {showReply ? (
-        <>
+        <WrapperRow style={{ marginTop: normalize(10) }}>
           <Input
             multiline
             minHeight={50}
             name="comment"
             label={texts.consul.commentLabel}
-            placeholder={texts.consul.comment}
+            placeholder={texts.consul.answer}
             autoCapitalize="none"
             control={control}
+            chat
           />
-          <WrapperVertical>
-            <Button
-              onPress={handleSubmit(onSubmit)}
-              title={
-                isLoading ? texts.consul.submittingCommentButton : texts.consul.commentAnswerButton
-              }
-              disabled={isLoading}
-            />
-          </WrapperVertical>
-        </>
+          <TouchableOpacity
+            onPress={handleSubmit(onSubmit)}
```

Comment:
i cannot click on the send icon for answers, if the keyboard is open. there is something missing on the view, that its pressable while keyboard is open i guess.

Reply:
i will look into this.

Reply:
adding `keyboardShouldPersistTaps` props to ListItem fixed the problem.

6d09d940ed07b93248d82864cc5fa3cd0d68ac79

Reply:
why have you used always there? what would other options be that react native provides us? have you checked the existing code base?

Reply:
yes, it doesn't need to be `always`. i will update it to `handled`.

Reply:
b725a4b6a93fddb8f9dde638db29c9c5fe0f8591

Reply:
do you know why? what is the difference in our case?

Reply:
i read the documentation and understood how it works.

- 'never', tapping outside of the focused text input when the keyboard is up dismisses the keyboard. When this happens, children won't receive the tap.
- 'always', the keyboard will not dismiss automatically, and the scroll view will not catch taps, but children of the scroll view can catch taps.
- 'handled', the keyboard will not dismiss automatically when the tap was handled by children of the scroll view (or captured by an ancestor).
- false, deprecated, use 'never' instead
- true, deprecated, use 'always' instead